cmd/aries-agentd/startcmd/start_test.go

```diff
@@ -0,0 +1,304 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package startcmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/spf13/cobra"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+)
+
+const testHostURL = "localhost:8080"
+const testInboundHostURL = "localhost:8081"
+
+func TestStartCmdContents(t *testing.T) {
+	startCmd := Cmd()
+
+	if startCmd.Use != "start" {
+		t.Error()
+	}
+	if startCmd.Short != "Start an agent" {
+		t.Error()
+	}
+	if startCmd.Long != `Start an Aries agent controller` {
+		t.Error()
+	}
+
+	if !areFlagPropertiesCorrect(startCmd, AgentHostFlagName, AgentHostFlagShorthand, AgentHostFlagUsage) {
```

Comment:
We won't know which of them are incorrect when this returns `false`. Rather just use `require.Equals()` within the body of `areFlagPropertiesCorrect()` - this way we can trace the failed assertion to an exact line of test code